packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -40,9 +40,13 @@ export interface Memory {
 export interface PerformanceWithMemory extends IsomorphicPerformance {
 	readonly memory: Memory;
 }
+
 /**
  * Broad classifications to be applied to individual properties as they're prepared to be logged to telemetry.
- * Please do not modify existing entries for backwards compatibility.
+ *
+ * @privateRemarks Please do not modify existing entries for backwards compatibility.
+ *
+ * @public
```

Comment:
Hmm this should probably be public.  Otherwise how will partners know what values to expect from FF (the `tag` property is merely `string`).

Reply:
Everything else in this file should be internal.

Reply:
We've decided to start with everything as `@internal` and do the exercise of promoting things used by Loop in a future pass. I've left this as `@internal` for now.